playbooks/adhoc/uninstall.yml

```diff
@@ -55,6 +55,10 @@
     - pcsd
     failed_when: false
 
+  - shell: "atomic uninstall atomic-openshift-master"
+    changed_when: False
```

Comment:
Instead of setting changed_when  to false, might be better to have failed when parse the rc or output indicates that a container was actually uninstalled.